packages/fbjs-scripts/gulp/check-dependencies.js

```diff
@@ -35,7 +35,10 @@ module.exports = function(opts) {
     });
 
     outdated.on('exit', function(code) {
-      if (code !== 0) {
+      // exitCode set to 1 in npm 4
+      if (code === 1) {
+        process.exitCode = 0
```

Comment:
Changing the exit code is unnecessary. 

I think a better approach is to move ```cb(new gutil.PluginError(PLUGIN_NAME, 'npm broke'));``` into the if statement that check ```failures.length ```. This way gulp only exits if packages are out of date. we ignore the exit code of npm outdated all together.